rezz.lic

```diff
@@ -1,118 +1,131 @@
-#   Documentation: https://elanthipedia.play.net/Lich_script_repository#rezz
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#rezz
+  
+  Define a waggle sets for rezz and rejuv if you don't want to use the default values in base-spells.yaml
```

Comment:
Can you move this to the elanthipedia documentation?

Reply:
There is no current elanthipedia documentation.  I'll create it there and remove it from here once we finalize how it's going to be setup here.